make-wheels.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+
+'''
+Author: Trishank Karthik Kuppusamy <trishank.kuppusamy@datadoghq.com>
+Date: Nov 27 2017
+Description: A simple Python script to build and test wheels.
+'''
+
+
+import os
+import subprocess
+
+
+def main():
+    # Get non-hidden directories.
+    dirs = sorted(name for name in os.listdir('.') \
```

Comment:
you might be able to get every dir you need with something like:
```
dirs = [os.path.dirname(d) for d in glob.glob('./**/setup.py')]
```
That could maybe simplify the logic in the for loop a little bit. This is fine, just a thought.